package.json

```diff
@@ -53,7 +53,6 @@
   ],
   "sideEffects": false,
   "scripts": {
-    "postinstall": "patch-package",
```

Comment:
Please also remove it from `devDependencies`.

Reply:
It is removed indeed!